impl.go

```diff
@@ -112,9 +125,19 @@ type Spec struct {
 
 // typeSpec locates the *ast.TypeSpec for type id in the import path.
 func typeSpec(path string, id string, srcDir string) (Pkg, Spec, error) {
-	pkg, err := build.Import(path, srcDir, 0)
-	if err != nil {
-		return Pkg{}, Spec{}, fmt.Errorf("couldn't find package %s: %v", path, err)
+	var pkg *build.Package
```

Comment:
Instead, how about:

```go
importFn := build.Import
if path == "" {
  // comment here explaining why, please
  importFn = build.ImportDir
}
pkg, err := importFn(path, srcDir, 0)
if err != nil {
  // as before
}
```

Reply:
`Import` and `ImportDir` take different numbers of parameters (`Import` takes 3, while `ImportDir` takes 2).  I could provide a wrapper or to curry the first argument to `Import`, but that wouldn't reduce the code much. If we do what you suggested here, then the error message for the `ImportDir` case would read strangely because the `path` value is empty.

Do you prefer to leave this as-is, wrap `Import`, or some other alternative?

Reply:
Oops. As-is, thanks.